docker/build/bin/km-build

```diff
@@ -8,14 +12,14 @@ set -vx
 GOPKG=github.com/mesosphere/kubernetes-mesos
 
 test ${#} -eq 0 && CMD=( make bootstrap install ) || CMD=( "${@}" )
-test -n "$GOPATH" || GOPATH=/pkg
+test -n "$GOPATH" || GOPATH=/go
```

Comment:
pretty sweet if that's the only place you need to change /pkg; what about the project-level Makefile? or hack/patches/apply.sh ?